model/fieldtypes/compat/Float.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
```

Comment:
Wrapping in a condition like that doesn't actually work. PHP 7 seems to parse the file and throw a "can't name class Float" error before even evaluating the condition.

What benefit does the new auto-loader add? Just means we don't have a conditional include of a stub? I suppose that makes sense.